apps/package.json

```diff
@@ -279,54 +206,128 @@
     "@typescript-eslint/eslint-plugin": "^5.57.0",
     "@typescript-eslint/parser": "^5.57.0",
```

Comment:
Should these two typescript linters be moved up?

Reply:
🦅 eyes ;-) I split this PR out of https://github.com/code-dot-org/code-dot-org/pull/56703, which was all about upgrading these (and I noticed they should be moved in the process, and one thing led to another...). I didn't move them here just because I moved them there, and didn't feel like a merge conflict.

Reply:
Just merged #56703